xds/src/main/java/io/grpc/xds/PriorityLoadBalancer.java

```diff
@@ -332,6 +329,12 @@ public void updateBalancingState(final ConnectivityState newState,
           seenReadyOrIdleSinceTransientFailure = false;
           failOverTimer.cancel();
         }
+
+        // If we are currently handling newly resolved addresses, let's nyt try to reconfigure as
```

Comment:
s/nyt/not/

s/will take care of that/will take care of that to provide an atomic config update/